tests/integration/docusaurus/connecting_to_your_data/filesystem/spark_yaml_example.py

```diff
@@ -92,8 +92,36 @@
 assert isinstance(validator, ge.validator.validator.Validator)
 assert [ds["name"] for ds in context.list_datasources()] == ["my_filesystem_datasource"]
 assert (
-    "yellow_trip_data_sample_2018-01"
+    "yellow_trip_data_sample_2019-01"
     in context.get_available_data_asset_names()["my_filesystem_datasource"][
         "default_inferred_data_connector_name"
     ]
 )
+
+
+# Here is a RuntimeBatchRequest using a path to a directory
+batch_request = RuntimeBatchRequest(
+    datasource_name="my_filesystem_datasource",
+    data_connector_name="default_runtime_data_connector_name",
+    data_asset_name="<YOUR_MEANINGFUL_NAME>",  # this can be anything that identifies this data_asset for you
+    runtime_parameters={"path": "<PATH_TO_YOUR_DATA_HERE>"},  # Add your path here.
+    batch_identifiers={"default_identifier_name": "something_something"},
```

Comment:
We had originally thought of {`default_identifier_name`: `default_identifier`} as a possibility, but was afraid that it would make it imply to users that the default shouldn't be changed.. with `something_something` the original intention was making it more "clear" that this was a value that should be changed. 

But coming back to it,  I can get behind the idea that it is a bit of an eyesore 😅 

What are your thoughts on `default_identifier` or `default_idenfitier_12345`?

Reply:
I like `default_identifier`!

Reply:
ok could we merge this PR as-is for this issue, and I'll do a subsequent PR for a global replace.